docs/source/dsa.md

```diff
@@ -0,0 +1,80 @@
+# Digital Slide Archive
+
+The [Digital Slide Archive][1] (DSA) is "a containerized web-based platform for the analysis, visualization, management and
+annotation of whole-slide digital pathology imaging data".
+It is an open-source project based on [Kitware][11]'s data management platform [Girder][10].
+
+## Azure deployment
+
+We have deployed the DSA on [Azure][2] to visualize our data and interpret our models and experiments.
+Below are instructions to replicate our deployment using your own data.
+
+### Host
+
+The first step is [creating a Linux virtual machine][4] (VM) to host the DSA.
+The code can be [downloaded from GitHub][5] and run using [Docker][6].
+By default, the application runs at port 8080.
+To use HTTPS, an SSL certificate must be obtained and port 443 may be specified instead within the Docker configuration.
+The DSA uses [Cherrypy][8] as the underlying server engine.
+Therefore, Cherrypy must be [configured][9] to use the SSL certificate installed in the VM.
+Ports in a VM can be opened using [network security group rules][7].
+
+### Storage
+
+The datasets we use are securely stored in [Azure Blob Storage][3] containers.
+Containers can be mounted on the host machine using [BlobFuse][13] and connected to Girder using [assetstores][12].
```

Comment:
Not clear to me. So I first have to mount the container, and then start assetstore? or does assetstore do that for me?

Reply:
Rephrased in 1d97167.